_data/changelogs/component-date-range-picker.yml

```diff
@@ -2,6 +2,14 @@ title: Date range picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused `mouseover` events to focus buttons and prevent keyboard navigation. 
+    summaryAdditional:  When you hover your mouse over the date picker buttons, only the hover state will be triggered.
```

Comment:
**suggestion:** Recommend we make this summary match date picker's
```suggestion
    summary: Fixed a bug that caused `mouseover` events to prevent keyboard navigation. 
    summaryAdditional:  Now when you hover your mouse over the date picker buttons, only the hover state will be triggered.
```